modules/govuk_jenkins/manifests/job/local_links_manager_import_service_interactions.pp

```diff
@@ -0,0 +1,25 @@
+# == Class: govuk_jenkins::job::local_links_manager_import_service_interactions
+#
+# Import services and interactions and the mapped service interactions into local-links-manager
+#
+class govuk_jenkins::job::local_links_manager_import_service_interactions (
+  $app_doman = hiera('app_doman'),
+) {
+
+  $check_name = 'local-links-manager-import-service-interactions'
+  $service_description = 'Import services and interactions into local-links-manager'
+  $job_url = "https://deploy.${app_domain}/job/local-links-manager-import-service-interactions/"
+
+  file { '/etc/jenkins_jobs/jobs/local_links_manager_import_service_interactions.yaml':
+    ensure  => present,
+    content => template('govuk_jenkins/jobs/local_links_manager_import_service_interactions.yaml.erb'),
+    notify  => Exec['jenkins_jobs_update'],
+  }
+
+  @@icinga::passive_check { "${check_name}_${::hostname}":
+    service_description => $service_description,
+    host_name           => $::fqdn,
+    freshness_threshold => 3600,
```

Comment:
If this only runs but once a month this will need to be higher (whatever a month is in seconds).